gateway/src/shard/impl.rs

```diff
@@ -703,6 +712,21 @@ impl Shard {
         }
     }
 
+    /// Get the current number of websocket messages allowed until the next rate limit reset,
+    /// excluding the number of heartbeats it takes for the shard to live.
+    ///
+    /// # Errors
+    ///
+    /// Returns a [`SessionInactiveError`] if the shard's session is inactive.
+    pub fn ratelimiter_tokens_left(&self) -> Result<u32, SessionInactiveError> {
```

Comment:
I'm not sure about this naming actually. It feels awkward. Maybe `command_ratelimit`, or `commands_remaining`? Or maybe just `ratelimit` -- short like [`latency`](https://docs.rs/twilight-gateway/latest/twilight_gateway/shard/struct.Information.html#method.latency)?